lib/cjs/generated/rum.d.ts

```diff
@@ -1604,6 +1614,46 @@ export interface ProfilingInternalContextSchema {
     readonly error_reason?: 'not-supported-by-browser' | 'failed-to-lazy-load' | 'missing-document-policy-header' | 'unexpected-exception';
     [k: string]: unknown;
 }
+/**
+ * Stream schema for media streaming properties
+ */
+export interface StreamSchema {
+    /**
+     * Stream properties
+     */
+    readonly stream?: {
+        /**
+         * current bitrate at the time of collection
+         */
+        bitrate?: number;
+        /**
+         * How long is the content (VOD only)
```

Comment:
Perhaps we should explicitly state what unit this duration property uses on this comment (and also on the stream schema below)

Reply:
it should be in milliseconds